app/assets/styles/user.scss

```diff
@@ -1,3 +1,49 @@
+$accent-color: #2C9FA3;
+$accent-color-light: #2DBDC2;
+$subheader-color: #AAAAAA;
+$font-choice: 'Nunito Sans', sans-serif;
+$transform: translate(-50%,0%);
+
 .editor-user {
-    word-wrap: break-word;
+  word-wrap: break-word;
+}
+
+#about, #suggest-resource, #contact, #login, #reset-password, #health, #rights {
+  margin-bottom: 50px;
+  position: absolute;
+  left: 50%;
+  top: 75px;
+  -webkit-transform: $transform;
+  -ms-transform: $transform;
+  transform: $transform;
+}
+
+#about, #health, #rights {
```

Comment:
same thing about adding making a general class instead of listing ids - maybe something relating to static pages like `.static-page`